runtime/empty.go

```diff
@@ -0,0 +1,233 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright Flow Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package runtime
+
+import (
+	"time"
+
+	"github.com/onflow/atree"
+	"go.opentelemetry.io/otel/attribute"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/cadence/runtime/interpreter"
+)
+
+type EmptyRuntimeInterface struct{}
```

Comment:
Maybe a comment on the intent/usage of this struct would be helpful.

Reply:
Added in [aef8f3d](https://github.com/onflow/cadence/pull/3485/commits/aef8f3d8f5c2937c1b20dd2f4ea80de12232a95a)